pkg/stargate/deployments.go

```diff
@@ -332,28 +338,62 @@ func computeHeapSize(template *api.StargateTemplate) resource.Quantity {
 	return resource.MustParse("256Mi")
 }
 
+// This config map will always be created by the k8ssandra controller.
```

Comment:
mmmmh, I didn't think of the case where Stargate could be used outside of K8ssandra.
My changes will prevent using the config map ref as expected because it's now used by the k8ssandra operator to generate the config map that will be mounted (which has a fixed name), using the referenced one.
Let me check how I can handle this better.

Reply:
This was moved to the Stargate controller now.